Revit_oM/Parameters/RevitIdentifiers.cs

```diff
@@ -21,21 +21,19 @@
  */
 
 using BH.oM.Base;
-using System;
-using System.Collections.Generic;
 using System.ComponentModel;
 
 namespace BH.oM.Adapters.Revit.Parameters
 {
     [Description("An entity containing identifiers of Revit element correspondent to the BHoM object that carries it.")]
-    public class RevitIdentifiers : IObject, IFragment, IImmutable
+    public class RevitIdentifiers : IPersistentAdapterId, IImmutable
     {
         /***************************************************/
         /****             Public Properties             ****/
         /***************************************************/
 
-        [Description("UniqueId of the Revit element correspondent to the BHoM object that carries this fragment.")]
-        public virtual string UniqueId { get; } = "";
+        [Description("PersistentId, known also as UniqueId of the Revit element correspondent to the BHoM object that carries this fragment.")]
```

Comment:
```suggestion
        [Description("The PersistentId is the UniqueId of the Revit element correspondent to the BHoM object that carries this fragment. This can be used for Diffing purposes.")]
```